### ISSUE TITLE
Card Info Sidebar can't work sometimes

### ISSUE THREAD

**Issue by purequant** [noobj2/Anki-Advanced-Review-Bottombar#10]
This addon is very good, but Card Info Sidebar can't work sometimes.

Display an error message:
```
Error
An error occurred. Please start Anki while holding down the shift key, which will temporarily disable the add-ons you have installed.
If the issue only occurs when add-ons are enabled, please use the Tools > Add-ons menu item to disable some add-ons and restart Anki, repeating until you discover the add-on that is causing the problem.
When you've discovered the add-on that is causing the problem, please report the issue on the add-on support site.
Debug info:
Anki 2.1.35 (84dcaa86) Python 3.8.0 Qt 5.14.2 PyQt 5.14.2
Platform: Windows 10
Flags: frz=True ao=True sv=1
Add-ons, last update check: 2020-11-09 11:16:17

Caught exception:
Traceback (most recent call last):
  File "D:\AnkiDataFolder\addons21\1136455830\Bottom_Bar.py", line 64, in <lambda>
    (info_shortcut, lambda: Card_Info._cs.toggle()),
  File "D:\AnkiDataFolder\addons21\1136455830\Card_Info.py", line 73, in toggle
    self.show()
  File "D:\AnkiDataFolder\addons21\1136455830\Card_Info.py", line 62, in show
    self._update()
  File "D:\AnkiDataFolder\addons21\1136455830\Card_Info.py", line 367, in _update
    txt += d.cardStats_mod(current_card)
  File "D:\AnkiDataFolder\addons21\1136455830\Card_Info.py", line 303, in cardStats_mod
    return CardStats(self, card).report_mod()
  File "D:\AnkiDataFolder\addons21\1136455830\Card_Info.py", line 223, in report_mod
    self.addLine("Created", time.strftime("%Y-%m-%d | %H:%M", time.localtime(c.id/1000)))
OSError: [Errno 22] Invalid argument

```

**Comment by noobj2** [comment]
can you give me some info to reproduce the error?

**Comment by purequant** [comment]
Actually, I don't know how to reproduce the error. If I restart Anki, Card Info Sidebar usually can work some time, then the error message pops up and it can't work anymore. Maybe it conflicts with other addons. Here is my addon list.

```
    
    Anki 2.1.35 (84dcaa86) Python 3.8.0 Qt 5.14.2 PyQt 5.14.2
    Platform: Windows 10
    Flags: frz=True ao=True sv=1
    Add-ons, last update check: 2020-11-09 11:16:17
    
    
    ===Add-ons (active)===
    (add-on provided name [Add-on folder, installed at, version, is config changed])
    Add Hyperlink ['318752047', 2020-07-30T04:37, 'None', mod]
    Additional Card Fields Fork for 21 ['744725736', 2020-09-20T19:50, 'None', mod]
    Advanced Browser ['874215009', 2020-11-08T19:03, 'None', '']
    Advanced Copy Fields ['1898445115', 2019-05-04T02:46, 'None', '']
    Advanced Review Bottom Bar ['1136455830', 2020-06-09T19:54, 'None', mod]
    Anki Persistence ['1051511270', 2020-09-02T03:32, 'None', '']
    Anki Simulator ['817108664', 2020-07-21T22:17, 'None', '']
    Anki Zoom ['538879081', 2020-04-04T08:20, 'v1.1.3', mod]
    AnkiConnect ['2055492159', 2020-11-02T08:16, 'None', '']
    Asset Manager ['656021484', 2020-10-25T08:04, 'None', '']
    Auto Markdown - fix version ['15061497', 2020-09-15T00:09, 'None', mod]
    Batch Editing ['291119185', 2019-08-02T13:00, 'None', '']
    Blitzkrieg II - Advanced Browser Sidebar BETA8 ['564851917', 2020-04-17T11:53, 'None', '']
    Browser Card Info Bar ['2140680811', 2020-11-03T19:27, 'None', mod]
    Browser MaximizeHide TableEditor ['1819291495', 2020-07-03T06:51, 'None', '']
    Browser TableEditor side-by-side horizontal split ['831846358', 2020-07-08T07:44, 'None', '']
    Change note type respect field and card names ['513858554', 2020-03-09T17:30, 'None', '']
    Clickable Tags v11 ['380714095', 2020-03-22T21:56, 'None', '']
    Closet For Anki ['272311064', 2020-11-05T09:37, 'None', '']
    Color Confirmation ['1084228676', 2020-03-14T20:13, 'None', '']
    Copy notes ['1566928056', 2020-08-03T00:21, 'None', mod]
    Correct Answers Graph ['993120073', 2020-03-24T20:57, 'None', '']
    Custom Styles font color background colour classes ['1899278645', 2020-09-20T20:23, 'None', '']
    Customize Keyboard Shortcuts ['0', 2020-09-02T12:53, 'None', mod]
    Customize Sidebar ['1988760596', 2020-01-16T05:01, 'None', '']
    Edit Field During Review ['1020366288', 2020-10-07T20:22, 'None', mod]
    Enhance main window ['877182321', 2020-09-03T05:49, 'None', mod]
    Extended Tag AddEdit Dialog ['1135507717', 2020-06-30T21:46, 'None', '']
    External Editor ['16727059', 2020-07-27T17:06, 'None', mod]
    External Note Editor for the Browser ['1996266159', 2020-04-01T04:59, 'None', '']
    Fast Word Query Multi-threaded queries for words from local or web dictionaries ['1807206748', 2020-01-06T19:45, 'None', '']
    Five Finger Silver New Answer Button Graph ReLrn Stats ReMemorize Plan9-0 ['2047355215', 2020-01-06T22:56, 'None', '']
    Flag label ['671965183', 2020-05-14T19:05, 'None', mod]
    Hierarchical Tags 2 ['594329229', 2020-04-12T00:36, 'None', '']
    High Yield Tags ['1918380616', 2020-07-01T04:35, 'None', '']
    Highlight Search Results in the Browser ['225180905', 2019-06-06T21:40, 'None', '']
    Keep model of add cards ['424778276', 2020-05-17T07:18, 'None', '']
    Leaderboard ['41708974', 2020-10-29T16:46, 'None', mod]
    Mini Format Pack ['295889520', 2019-06-06T21:37, 'None', mod]
    More Decks Stats and Time Left ['1556734708', 2020-04-01T04:19, 'None', '']
    More Overview Stats 21 ['738807903', 2019-06-06T22:32, 'None', mod]
    Open linked pdf docx epub audiovideo etc in external Program ['879473266', 2020-07-20T17:19, 'None', mod]
    OpenInExternalEditorRenameDuplicate for ImageAudioVideo ['1560623518', 2020-08-06T17:28, 'None', '']
    Opening the same window multiple time ['354407385', 2020-05-16T08:59, 'None', '']
    Pretzel Logic Retention Benchmarking ['1168065472', 2020-04-25T11:13, 'None', mod]
    Progress Graphs and Stats for Learned and Matured Cards ['266436365', 2020-03-29T14:26, 'None', '']
    Quick Field Navigation ['734297936', 2019-12-30T10:24, 'None', '']
    Quick note and deck buttons Fork for 21 ['1331140127', 2020-07-14T02:54, 'None', mod]
    Quick tagging 21 ['304770511', 2020-02-11T21:09, 'None', mod]
    Refocus Card when Reviewing 21 ['1642550423', 2018-09-19T22:14, 'None', '']
    Remaining time for Anki 21 ['1508357010', 2020-10-21T23:02, '20.10.21i160', '']
    Remove Cloze Button and Hotkey ['836994508', 2019-12-22T08:49, 'None', mod]
    Remove Duplicate Cards ['95590040', 2020-04-25T14:55, 'None', '']
    Remove card history ['2089200096', 2020-01-03T15:16, 'None', '']
    Review Heatmap ['review_heatmap', 2020-04-30T21:23, 'None', '']
    Reviewer Context Menu ['281854631', 2019-08-09T14:09, 'None', '']
    Session Timer ['1287298831', 2020-07-01T00:35, 'None', '']
    Showing time in browser ['1243668133', 2020-03-11T06:13, 'None', '']
    Small Potatoes Show learning step count in days ['75718778', 2020-04-03T02:53, 'None', '']
    Speed Focus Mode auto-alert auto-reveal auto-fail ['1046608507', 2019-06-06T18:27, 'None', '']
    Statistics for Note Types Card Types and Tags ['16703369', 2020-07-20T10:11, 'None', '']
    Syllabus - Anki Statistics and Export by Tag and Deck ['668156440', 2020-03-05T14:18, 'None', mod]
    Syntax Highlighting Enhanced Fork ['1972239816', 2020-09-13T19:48, 'None', mod]
    TagSelectorV2 ['1022577188', 2020-08-13T11:09, 'None', '']
    Time Spent Creating Cards ['1178407876', 2020-07-02T04:08, 'None', '']
    Touch Wood Show New Cards Left Hint ['1498273991', 2020-04-04T12:37, 'None', '']
    Web Browser - Search terms Import texts and images automatically ['864545277', 2020-08-30T02:23, 'None', '']
    Ze Add Note Id ['886709118', 2020-07-31T00:12, 'None', '']
    Ze Frozen Fields ['94610912', 2020-09-09T19:26, 'None', '']
    add-on dialog searchfilter bar ['561945101', 2020-05-08T21:16, 'None', '']
    copy note contents to new Add window from Browser Reviewer Add EditCurrent ['759018284', 2020-01-07T19:37, 'None', mod]
    editor paste raw unfiltered full html ['1765712663', 2020-08-05T03:48, 'None', '']
    extended html editor for fields and card templates with some versioning ['1043915942', 2020-07-30T01:19, 'None', mod]
    mergenotes ['1425504015', 2020-09-11T11:14, 'None', '']
    paste plainunformatted text ['107041104', 2020-11-08T21:24, 'None', mod]
    
    ===IDs of active AnkiWeb add-ons===
    1020366288 1022577188 1043915942 1046608507 1051511270 107041104 1084228676 1135507717 1136455830 1168065472 1178407876 1243668133 1287298831 1331140127 1425504015 1498273991 15061497 1508357010 1556734708 1560623518 1566928056 1642550423 16703369 16727059 1765712663 1807206748 1819291495 1898445115 1899278645 1918380616 1972239816 1988760596 1996266159 2047355215 2055492159 2089200096 2140680811 225180905 266436365 272311064 281854631 291119185 295889520 304770511 318752047 354407385 380714095 41708974 424778276 513858554 538879081 561945101 564851917 594329229 656021484 668156440 671965183 734297936 738807903 744725736 75718778 759018284 817108664 831846358 836994508 864545277 874215009 877182321 879473266 886709118 94610912 95590040 993120073
    
    ===Add-ons (inactive)===
    (add-on provided name [Add-on folder, installed at, version, is config changed])
    Add tagmove card in review ['647335063', 2020-06-12T18:32, 'None', '']
    Ankdown ['109255569', 2019-07-19T01:17, 'None', mod]
    AnkiWebView Inspector ['31746032', 2019-08-02T13:41, 'None', '']
    Ankidown - The advanced importer for Anki ['38786043', 2020-04-16T23:56, 'None', '']
    AwesomeTTS Google Cloud Text-to-Speech unofficial ['814349176', 2020-10-20T18:40, 'None', '']
    Basic Printing Support ['1025789669', 2020-02-24T14:12, 'None', mod]
    BetterSearch ['1052724801', 2020-10-26T04:49, 'None', '']
    Cardistry 2 Dynamically Adjust New Cards ['1535078906', 2020-09-04T04:52, 'None', mod]
    Edit Field During Review Cloze ['385888438', 2020-06-11T20:32, 'None', mod]
    Export preview to HTML ['96435883', 2019-05-31T20:06, 'None', '']
    Fast cards reposition ['544982740', 2019-06-10T22:52, 'None', '']
    Field History ['1247884413', 2020-01-07T21:24, 'None', mod]
    Fill the blanks - Multiple typecloze support ['1933645497', 2020-03-14T09:20, 'None', '']
    Flexible Duplicate Checking ['1587955871', 2020-05-19T14:49, 'None', '']
    HTML Window source ['1214415810', 2020-02-18T21:20, 'None', '']
    History Recorder ['2039239436', 2020-08-23T21:26, 'None', '']
    HoochieMama Randomize Rev Queue ['1460733408', 2020-06-03T21:17, 'None', '']
    Image Occlusion Enhanced for Anki 21 alpha ['1374772155', 2020-04-28T10:21, 'None', '']
    Knockout Deck Subset Learning ['899747926', 2020-01-05T20:20, 'None', '']
    LPCG LyricsPoetry Cloze Generator ['2084557901', 2020-07-12T07:06, 'None', '']
    Media Import ['1531997860', 2019-08-20T22:25, 'None', '']
    Merge two notes into a single note ['1482505492', 2020-05-28T11:48, 'None', '']
    No Distractions Full Screen Clean review interface  TabletTouch support ['1049863218', 2020-08-14T06:59, 'v4.1.8', mod]
    Postpone cards review ['1152543397', 2020-05-29T11:40, 'None', '']
    ReMemorize Rescheduler with sibling and logging v140 ['323586997', 2020-04-17T15:07, 'None', '']
    Reviewer to Browser choosing what to display ['1555020859', 2020-03-12T12:48, 'None', '']
    Search Bar ['1251668918', 2020-09-10T05:55, 'None', '']
    Searching PDF Reading  Note-Taking in Add Dialog ['1781298089', 2020-11-07T22:43, 'None', mod]
    Straight Reward ['957961234', 2020-10-08T23:37, 'None', '']
    Study Logger ['1919929679', 2020-04-02T11:54, 'None', '']
    Trigger and actions change one card depending on what occurs on another card ['1981494159', 2020-05-05T04:53, 'None', mod]
    Vocabulary Extractor study difficult words prior to reading a book ['1152823001', 2020-09-24T21:56, 'None', mod]
    ankiboards ['1350713399', 2020-10-14T12:45, 'None', '']
    export all notetypesmodels and deckoptionsgroups without notes or media ['870620435', 2020-01-15T22:47, 'None', '']
    extended editor for field for tables searchreplace  ['805891399', 2020-07-30T06:54, 'None', '']
    link Cards Notes and Preview them in extra window ['1423933177', 2020-10-17T19:57, 'None', '']
    load balancer ['1417170896', 2020-08-06T07:58, 'None', '']
    put ALL due learning cards first modified v2 support ['1807121582', 2020-01-07T21:20, 'None', '']
    remove linebreaks  do word wrap fork for 21 ['2028179508', 2020-05-11T20:58, 'None', '']

```

**Comment by noobj2** [comment]
how often does that happen? does it happen out of nowhere or does it happen on a specific deck/note type? what about the review buttons? does it usually happen when you press a specific review button or it doesn't make any difference.
do you have the sidebar open all the time? or you open/close it? does it happen when you try to open the sidebar or it happens when it's already open?
OR it happens totally randomly?
